t/FHEM/00_SIGNALduino/01_SIGNALduino_Parse_MU.t

```diff
@@ -15,56 +15,63 @@ my @mockData = (
         plan => 2,
         testname =>  q[Corrupt Mu data, combined message],
         input =>  q[MU;P0=-2272;P1=228;P2=-356;P3=635;P4=-562;P5=433;D=012345234345252343452523434345252345234343434523434345252343452525252525234523452343452345252525;CP=5;R=4�;P3=;L=L=-2864;L=H=2980;S=L=-1444;S=H=1509;D=354146333737463037;C==1466;L==32;R==9;],
-        rValue => U(), 
-        todoReason => q[This data should not be processed]
+        rValue => U(),
     },
     {
         deviceName => q[dummyDuino],
         plan => 2,
         testname =>  q[Corrupt MU data, unknown specifier V=],
         input =>  q[MU;P0=-1440;P1=432;P2=-357;P3=635;P4=-559;D=012121212123412343412123434121234343412123412343434341234343412123434121212121212341231212343412341212121;CP=1;V=139;],
```

Comment:
@HomeAutoUser

Das ist die eine rmsg welche nun nicht mehr dazu führt dass parse in FS10 aufgerufen wird